src/System.Linq.Expressions/src/System/Linq/Expressions/ExpressionStringBuilder.cs

```diff
@@ -217,94 +216,54 @@ protected internal override Expression VisitBinary(BinaryExpression node)
+                    case ExpressionType.GreaterThanOrEqual:    op = ">=";                        break;
+                    case ExpressionType.LessThanOrEqual:       op = "<=";                        break;
+                    case ExpressionType.Add:
+                    case ExpressionType.AddChecked:            op = "+";                         break;
+                    case ExpressionType.AddAssign:
+                    case ExpressionType.AddAssignChecked:      op = "+=";                        break;
+                    case ExpressionType.Subtract:
+                    case ExpressionType.SubtractChecked:       op = "-";                         break;
+                    case ExpressionType.SubtractAssign:
+                    case ExpressionType.SubtractAssignChecked: op = "-=";                        break;
+                    case ExpressionType.Divide:                op = "/";                         break;
+                    case ExpressionType.DivideAssign:          op = "/=";                        break;
+                    case ExpressionType.Modulo:                op = "%";                         break;
+                    case ExpressionType.ModuloAssign:          op = "%=";                        break;
+                    case ExpressionType.Multiply:
+                    case ExpressionType.MultiplyChecked:       op = "*";                         break;
+                    case ExpressionType.MultiplyAssign:
+                    case ExpressionType.MultiplyAssignChecked: op = "*=";                        break;
+                    case ExpressionType.LeftShift:             op = "<<";                        break;
+                    case ExpressionType.LeftShiftAssign:       op = "<<=";                       break;
+                    case ExpressionType.RightShift:            op = ">>";                        break;
+                    case ExpressionType.RightShiftAssign:      op = ">>=";                       break;
+                    case ExpressionType.And:                   op = IsBool(node) ? "And" : "&";  break;
+                    case ExpressionType.AndAssign:             op = IsBool(node) ? "&&=" : "&="; break;
+                    case ExpressionType.Or:                    op = IsBool(node) ? "Or"  : "|";  break;
+                    case ExpressionType.OrAssign:              op = IsBool(node) ? "||=" : "|="; break;
+                    case ExpressionType.ExclusiveOr:           op = "^";                         break;
+                    case ExpressionType.ExclusiveOrAssign:     op = "^=";                        break;
+                    case ExpressionType.Power:                 op = "^";                         break;
+                    case ExpressionType.PowerAssign:           op = "**=";                       break;
+                    case ExpressionType.Coalesce:              op = "??";                        break;
```

Comment:
Alas, we don't have a `switch` or `match` expression yet, so we need these tedious `break` statements. I've kept the order the same compared to the original, but this should make it easier to match columns in a glimpse. The only change is the conditional in `And`, `Or`, and the compound assignment variants.